src/engine.js

```diff
@@ -143,33 +150,79 @@ Engine.prototype.off = function (event, callback, context) {
  *
  */
 Engine.prototype.reload = function (options) {
+  options = options || {};
+  // Using a debouncer to optimize consecutive calls to reload the map.
+  // This allows to change multiple map parameters reloading the map only once,
+  // and therefore avoid the "You are over platform's limits" Windshaft error.
   return new Promise(function (resolve, reject) {
-    var self = this;
-    options = this._buildOptions(options);
-    var successCb = options.success;
-    var errorCb = options.error;
-    options.success = function (serverResponse) {
-      successCb(serverResponse);
-      resolve();
-    };
-    options.error = function (errors) {
-      errorCb(errors);
-      var error = self._getSimpleWindshaftError(errors);
-      reject(error);
+    batchOptions = {
```

Comment:
About this options:
- sourceId: ok. It should have the latest option, if any.
- forceFetch: this should be `true` if any stack call is `true`. But this condition is wrong, because if `batchOptions.forceFetch` is false and `options.forceFetch` is true, it returns false.
- includeFilters should have the last includeFilters. In this condition, it's using the last saved options if they exist.

Reply:
Yes, I left that pending for discussion.
- sourceId: latest
- forceFetch: OR-ed flags
- includeFilters: I'm not sure about this options. By default is true, so if you want to disable it all should be false, true otherwise (?)